docbook/src/docbkx/autoscale-devguide.xml

```diff
@@ -2201,6 +2229,20 @@ config at https://github.com/rackerlabs/autoscaling-chef/blob/master/site-cookbo
                 server is, and which cloud load balancer or
                 RackConnectV3 load balancer pool the new server
                 connects to.</para>
+            <note>
+                <itemizedlist>
+                    <listitem><para>
+                        The <parameter>launchConfiguration</parameter> uses the <emphasis role="bold">admin</emphasis> user’s SSH key pair name, 
```

Comment:
Maybe "The launchConfiguration uses the admin user to scale up, usually the first admin user found on the tenant.  Only that particular admin user's SSH key pair names can be used in the launchConfiguration."  ?

The reason being that the key pair name only matters if the user specifies a key name in the launch configuration - if it's unspecified, Cloud Servers does not do anything with keys and hence the key pair names do not matter.